src/api/spec/bootstrap/features/webui/interconnects_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'browser_helper'
```

Comment:
This file looks like a copy and paste of the non-Bootstrap specs. We still need to have specs covering the Bootstrap views.

There is also another spec which we previously [skipped](https://github.com/openSUSE/open-build-service/pull/7015/files#diff-06ada177f6adc513be0320acb26e69acR20), we need to enable it again.

Reply:
@dmarcoux 

> This file looks like a copy and paste of the non-Bootstrap specs. We still need to have specs covering the Bootstrap views.

I am not sure what you mean, can you please elaborate?

Reply:
@dmarcoux 

> This file looks like a copy and paste of the non-Bootstrap specs. We still need to have specs covering the Bootstrap views.

I am not sure what you mean, can you please elaborate?

Reply:
This file is exactly the same as `spec/features/webui/interconnects_spec.rb`, it's a copy and paste. I noticed because we still have the `skip_if_bootstrap` on lines 7 and 23.

Reply:
you mean that we could enable it in the other one instead of creating this? If that is what you mean, I agree

Reply:
@dmarcoux there is no file https://github.com/openSUSE/open-build-service/tree/master/src/api/spec/features/webui/interconnects_spec.rb :confused:

Reply:
It's in this PR, check this link: https://github.com/openSUSE/open-build-service/pull/7034/files#diff-a38b07e9016f25d4c1ff4c2d16ac8a1c

Reply:
got it now

Reply:
I get it now, thanks!